modules/infra/api/gr_infra.h

```diff
@@ -193,6 +193,20 @@ struct gr_infra_iface_set_req {
 	uint64_t set_attrs;
 };
 
+#define GR_INFRA_IFACE_STATS_GET REQUEST_TYPE(GR_INFRA_MODULE, 0x0006)
+
+struct gr_infra_iface_stats_get_req { };
+
+struct gr_iface_stats {
+	char name[GR_IFACE_NAME_SIZE + 16];
+	uint64_t value;
```

Comment:
The number of statistics is fixed here and we know them in advance. Instead of exporting strings, please export something that looks like the iface_stats structure in gr_iface.h.